locations/views.py

```diff
@@ -37,8 +38,23 @@ class CreateLocation(LoginRequiredMixin, AjaxTemplateMixin, CreateView):
     template_name = 'locations/create.html'
     ajax_template_name = 'locations/create_inner.html'
 
+    def get_context_data(self, **kwargs):
+        data = super(CreateLocation, self).get_context_data(**kwargs)
```

Comment:
With Python 3, we can shorten the `super()` call
```suggestion
        data = super().get_context_data(**kwargs)
```